qiskit/algorithms/evolvers/trotterization/trotter_qrte.py

```diff
@@ -50,18 +50,23 @@ class TrotterQRTE(RealEvolver):
 
         .. jupyter-execute::
 
+            import warnings
             from qiskit.opflow import X, Z, Zero
             from qiskit.algorithms import EvolutionProblem, TrotterQRTE
             from qiskit import BasicAer
             from qiskit.utils import QuantumInstance
 
-            operator = X + Z
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                operator = X + Z
             initial_state = Zero
             time = 1
             evolution_problem = EvolutionProblem(operator, 1, initial_state)
             # LieTrotter with 1 rep
             backend = BasicAer.get_backend("statevector_simulator")
-            quantum_instance = QuantumInstance(backend=backend)
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                quantum_instance = QuantumInstance(backend=backend)
```

Comment:
This PR doesn't address the modifications in all the rest of the code to remove opflow and QuantumInstance. This should be done in one or more separate PRs since, as you noted, is massive. I will change this PR to draft for now.